src/Sarif.FunctionalTests/v2/ConverterTestData/PREfast/AA_IndexAlias.cpp.xml.sarif

```diff
@@ -1,6 +1,6 @@
 {
-  "$schema": "http://json.schemastore.org/sarif-2.0.0",
```

Comment:
>json [](start = 21, length = 4)

you can't check in these updates, as they have improperly stripped information we expect in the test files. in order for updating to work properly, you need to enlist in the d: drive, have you done so locally? otherwise the tools can't locate the appropriate files to inject.

note that we could make this test code more resilient by converting these file locations and using uriBaseIds. the rewriters can accept arbitrary values for the uri base id values. this rewrite always injects the non-deterministic uri base id into the rewritten file as part of originalUriBaseIds, however. would be useful to make this behavior configurable, so that we could avoid injecting it into test content. another way to do so would be to write the 'remove non-deterministic content' rewrite feature which has been open for a while.

Reply:
Running `UpdateBaseline.ps1` in an enlistment rooted in `D:\src\sarif-sdk` restored the `region` and `snippet` properties.

---
In reply to: [221754585](https://github.com/Microsoft/sarif-sdk/pull/1105#discussion_r221754585) [](ancestors = 221754585)

Reply:
... but it does _not_ restore the `content` and `hashes` properties in the `files` dictionary (see `Sarif.FunctionalTests\v2\ConverterTestData\FxCop\SarifSdkTests.xml.sarif. I still think this might be due to the recent dictionary visitor work. @michaelcfanning.

---
In reply to: [222757596](https://github.com/Microsoft/sarif-sdk/pull/1105#discussion_r222757596) [](ancestors = 222757596,221754585)

Reply:
As I just mailed you: it's because that file was generated in an enlistment rooted at `E:/src/sarif-sdk-test`, not `D:/src/sarif-sdk`. I'm going to fix that.

---
In reply to: [222761104](https://github.com/Microsoft/sarif-sdk/pull/1105#discussion_r222761104) [](ancestors = 222761104,222757596,221754585)

Reply:
Fixed.

---
In reply to: [222785738](https://github.com/Microsoft/sarif-sdk/pull/1105#discussion_r222785738) [](ancestors = 222785738,222761104,222757596,221754585)